### ISSUE TITLE
Error installing on php 8.1

### ISSUE THREAD

**Issue by shadracnicholas** [mettle/sendportal-core#153]
I get the below erro when try to use the package.
 
 Environment - php8.1, Larevel 8.75

Error:

Problem 1
    - mettle/sendportal-core[v2.0.2, ..., v2.0.3] require kriswallsmith/buzz ^1.2 -> satisfiable by kriswallsmith/buzz[1.2.0].
    - mettle/sendportal-core[v2.0.0, ..., v2.0.1] require php ^7.3 -> your php version (8.1.0) does not satisfy that requirement.
    - kriswallsmith/buzz 1.2.0 requires symfony/options-resolver ^3.4 || ^4.0 || ^5.0 -> found symfony/options-resolver[v3.4.0-BETA1, ..., 3.4.x-dev, v4.0.0-BETA1, ..., 4.4.x-dev, v5.0.0-BETA1, ..., 5.4.x-dev] but the package is fixed to v6.0.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
    - Root composer.json requires mettle/sendportal-core ^2.0 -> satisfiable by mettle/sendportal-core[v2.0.0, v2.0.1, v2.0.2, v2.0.3].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.

**Comment by awizemann** [comment]
I also get this issue with php 8.1 and laravel 9. Please update the dependency mapping for php 8.

**Comment by soixt** [comment]
Same here on laravel 9 and php 8.1..

**Comment by Kayrah87** [comment]
Same here on laravel 9 and php 8.0

**Comment by HeadStudios** [comment]
Yep.... same here... wth. What do we do? Do we downgrade the PHP version?

**Comment by mxaadh** [comment]
Same
